### PR TITLE
feat: new website

### PR DESCRIPTION
**Pull request by dcposch**
## Screenshot

localhost_3001_ (1)

## Tasks

- [x] Basic page
- [x] Anchor links
- [x] Mobile
- [x] Team social links
- [x] Figure out what to do for the testimonial

### CODE REVIEW

**Comment by dcposch** [comment]
## Desktop

localhost_3001_ (2)

## Mobile

localhost_3001_(iPhone SE)

**Comment by dcposch** [comment]
### Testimonial

image

File: apps/daimo-web/public/assets/icon-global.png
Comment (nalinbhardwaj) [inline apps/daimo-web/public/assets/icon-global.png:1]: can delete these old icons? (I think theyre unused now)

File: apps/daimo-web/src/app/SectionFAQ.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+}
+
+function Question({
+  question,
+  answerHtml,
+}: {
+  question: string;
+  answerHtml: string;
+}) {
+  // const [expanded, setExpanded] = useState(false);
```
Comment (nalinbhardwaj) [inline apps/daimo-web/src/app/SectionFAQ.tsx:38]: can delete?

File: apps/daimo-web/src/app/SectionHero.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+          <div className="h-6 md:h-12" />
+          <DownloadLinkButton />
+        </div>
+        <div className="basis-0 flex-grow relative">
+          <Image
+            src="/assets/landing-screenshots.png"
+            width={1154}
+            height={750}
+            alt="Screenshots"
+            className="h-[32rem] w-[49.28rem] max-w-none absolute"
```
Comment (nalinbhardwaj) [inline apps/daimo-web/src/app/SectionHero.tsx:23]: `49.28rem` seems like a very specific number, maybe round it if not particularly chosen for some reason?

File: apps/daimo-web/src/components/DownloadLink.tsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    >
+      {title}
+    </Link>
+  );
+}
+
+function useDownloadTitleLink() {
+  const [[title, link], setTitleLink] = useState([
+    "Download on App Store",
+    "https://testflight.apple.com/join/j3ixWtuN",
```
Comment (nalinbhardwaj) [inline apps/daimo-web/src/components/DownloadLink.tsx:35]: do we want to start linking the testflight/play store directly already? can link https://noteforms.com/forms/daimo-uk2fe4?notionforms=1&utm_source=notionforms in the short term and switch when we're mainnet-ready

**Review by nalinbhardwaj — APPROVED** [review]
looks great

File: apps/daimo-web/src/app/SectionHero.tsx (hunk 2 above)
Comment (dcposch) [reply apps/daimo-web/src/app/SectionHero.tsx:23]: 32 / (aspect ratio of image)

**Pull request merged**